tests/models/test_right_model.py

```diff
@@ -25,5 +23,5 @@ def test_right_model(seed_everything, data_dir, frames_per_sample, frames_to_dro
     model = DirectRightModel(frames_per_sample, frames_to_drop, is_color_input, is_color_output)
 
     # train
-    trainer = pl.Trainer(fast_dev_run=True, callbacks=[RightCallback(dm.vis_img_dataloader()),FidCallback(dm.train_dataloader(),dm.val_dataloader())])
-    trainer.fit(model, dm.train_dataloader(), dm.val_dataloader())
+    trainer = pl.Trainer(fast_dev_run=True, callbacks=[RightCallback()])
```

Comment:
Is it possible to keep the FidCallback?

Reply:
it will work but it slows things down a lot so figured i want to run these models now and add back fid when we've figure it out

Reply:
i dont think we should run with fidcallback now if its just going to slow down models - i could run these without fid and run one with but my plan was to run without and then look at fid separately?